little_timmy/config_loader.py

```diff
@@ -189,21 +200,48 @@ def setup_run(root_dir: str, absolute_path: str = "") -> Context:
     # Setup dataloader and vault
     loader = DataLoader()
     vault_ids = C.DEFAULT_VAULT_IDENTITY_LIST
-    vault_secrets = cli.CLI.setup_vault_secrets(loader, vault_ids=vault_ids)
+    
+    # In ansible >= 12, VaultSecretsContext can only be initialized once
```

Comment:
would it work if it's initialised only once for every version of ansible to simplify the checks?